frontend/GoodGrocer/app/components/RequestCard.js

```diff
@@ -1,58 +1,79 @@
 import React from "react";
 import { Image, Text, View, StyleSheet, TouchableOpacity } from "react-native";
-import { Card, Button, Title, Paragraph } from "react-native-paper";
-import { Dim, Colors, Font, BorderRadius } from "../Constants";
+import { Ionicons } from "@expo/vector-icons";
+import { Dim, Colors, Font } from "../Constants";
 
 const RequestCard = (props) => {
   return (
-    <TouchableOpacity onPress={props.onPress}>
-      <Card style={Styles.container}>
-        <View style={Styles.user}>
-          <Card.Content>
-            <Image
-              source={{
-                uri: props.imageUri,
-              }}
-              style={Styles.image}
-            />
-            <Title>{props.name}</Title>
-          </Card.Content>
-          <Card.Content style={Styles.text}>
-            <Text>
-              <Paragraph>Store: </Paragraph>
-              <Text>{props.storeName}</Text>
-            </Text>
-            <Text>
-              <Paragraph>Items: </Paragraph>
-              <Text>{props.numItems}</Text>
-            </Text>
-            {props.requestComplete ? (
-              <Text style={Styles.completeText}>Complete</Text>
-            ) : null}
```

Comment:
was also using this component for the active errand screen, I can either remake a component for just the active errand like i did for the active item since there needs to be more info or we should add this somewhere in your new design where it looks good

Reply:
ohh wait i never mind, i read more of the code and saw that you added in the complete thing a different way